### PR TITLE
regen: dummy dcamera support

### PR DESCRIPTION
**Pull request by fredyshox**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
This seems like a lot of code (in multiple places) to send dummy frames.

**Comment by fredyshox** [comment]
it would have been shorter (just DummyFrameReader), but discovered that dmonitoringmodeld does not support non-tici frames anymore, so dummy frames are required for c2 segment regen - so it needs to be explicitely detected and handled...

**Comment by fredyshox** [comment]
we will phase out c2 segments at some point, but there's still significant amount of them in our training routes, so I would keep some basic support for them in process_replay stack

**Comment by fredyshox** [comment]
Simplified it a little bit. now usage of dummy dcamera is completely dependent on `dummy_dcamera` param

File: selfdrive/test/process_replay/test_processes.py [hunk 1]
```diff
@@ -41,23 +41,23 @@
...
-  ("CHRYSLER", "aregen99B094E1E2E|2023-05-10--14-41-40--0"),
-  ("RAM", "aregen5C2487E1EEB|2023-05-10--14-44-09--0"),
-  ("SUBARU", "aregen98D277B792E|2023-05-10--14-46-46--0"),
-  ("GM", "aregen377BA28D848|2023-05-10--14-48-28--0"),
-  ("GM2", "aregen7CA0CC0F0C2|2023-05-10--14-51-00--0"),
-  ("NISSAN", "aregen7097BF01563|2023-05-10--14-52-43--0"),
-  ("VOLKSWAGEN", "aregen765AF3D2CB5|2023-05-10--14-54-23--0"),
-  ("MAZDA", "aregen3053762FF2E|2023-05-10--14-56-53--0"),
-  ("FORD", "aregenDDE0F89FA1E|2023-05-10--14-59-26--0"),
+  ("BODY", "regen7FE9F3C7CE3|2023-10-25--23-56-32--0"),
```
Comment (adeebshihadeh) [inline selfdrive/test/process_replay/test_processes.py:44]: updating segments should be split from the new functionality

**Pull request merged**